frontend/src/components/investigate/InvestigateGraph.css

```
@@ -158,11 +160,66 @@
+.investigate-node:focus-visible .investigate-node-hit {
+  stroke: var(--focus-ring, var(--accent-primary));
+  stroke-width: 2;
+  fill: transparent;
+}
+
+.investigate-live {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)